tests/config/driverconfig/powerflex/v2.4.0/controller.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: <DriverDefaultReleaseName>-controller
-  namespace: <DriverDefaultReleaseNamespace>
```

Comment:
@VamsiSiddu-7 , why are we adding the set of yaml files for each of the supported versions under tests. I believe this was working with the one version. Also these are repetitions of yamls under driver config. Please keep only one version and remove the rest..

Reply:
done